druid-mr/src/main/java/com/yahoo/druid/hadoop/DruidHelper.java

```diff
@@ -40,23 +36,16 @@
   public List<String> getSegmentPathsToLoad(final String dataSource, final Interval interval,
       final String storageDir, final String overlordUrl) {
     List<DataSegment> segmentsToLoad = getSegmentsToLoad(dataSource, interval, overlordUrl);
-    VersionedIntervalTimeline<String, DataSegment> timeline = new VersionedIntervalTimeline<String, DataSegment>(
-          Ordering.<String>natural().nullsFirst()
-      );
-    for (DataSegment segment : segmentsToLoad) {
-      timeline.add(segment.getInterval(), segment.getVersion(), segment.getShardSpec().createChunk(segment));
-    }
-    final List<TimelineObjectHolder<String, DataSegment>> timeLineSegments = timeline.lookup(interval);
```

Comment:
you are right, thanks. I guess, it wasn't there in 0.6 and that is why I probably added it here.